db/migrations/versions/004_alter_field_nullabilities_for_round_1_data.py

```diff
@@ -0,0 +1,41 @@
+"""empty message
+
+Revision ID: 3c57bc0427b8
```

Comment:
just checking, is this id still right? Did you re-generate the migration or just change the name of the existing one? 

I think this line needs to reflect the file name, but the old auto-generated ones here don't seem to account for the date prefix so just checking it's not still using that hash id.